site/src/content/docs/forms/checkbox.mdx

```diff
@@ -110,217 +144,244 @@ You can display an icon by adding `.control-item-assets-container` with an icon
 
 ### Helper text
 
-You can display an helper text by adding a `.control-item-helper` as a sibling of a `.control-item-label`, don’t forget to make it accessible by adding an `aria-describedby` attribute on the input.
+You can display a helper text by adding a `.control-item-helper` as a sibling of a `.control-item-label`. Make sure the helper text is accessible by adding an `aria-describedby` attribute to the input.
 
 <Example code={`<div class="checkbox-item">
     <div class="control-item-assets-container">
-      <input class="control-item-indicator" type="checkbox" aria-describedby="checkboxHelperTextDescription" id="checkboxHelperText" value="" checked />
+      <input class="control-item-indicator" type="checkbox" aria-describedby="checkboxHelperTextDescription" id="checkboxHelperText" value="" />
     </div>
     <div class="control-item-text-container">
-      <label class="control-item-label" for="checkboxHelperText">Label</label>
-      <p class="control-item-helper" id="checkboxHelperTextDescription">Helper Text</p>
+      <label class="control-item-label" for="checkboxHelperText">Default checkbox</label>
+      <p class="control-item-helper" id="checkboxHelperTextDescription">Helper text</p>
     </div>
   </div>
   <div class="checkbox-item">
     <div class="control-item-assets-container">
-      <input class="control-item-indicator" type="checkbox" aria-describedby="checkboxHelperTextDescription2" id="checkboxHelperText2" value="" />
+      <input class="control-item-indicator" type="checkbox" aria-describedby="checkboxHelperTextDescription2" id="checkboxHelperText2" value="" checked />
     </div>
     <div class="control-item-text-container">
-      <label class="control-item-label" for="checkboxHelperText2">Label</label>
-      <p class="control-item-helper" id="checkboxHelperTextDescription2">Helper Text</p>
+      <label class="control-item-label" for="checkboxHelperText2">Checked checkbox</label>
+      <p class="control-item-helper" id="checkboxHelperTextDescription2">Helper text</p>
     </div>
   </div>`} />
 
-### Reverse
+## Layout
 
-You can reverse the component by adding `.control-item-reverse` to a `.checkbox-item`.
+### Default
```

Comment:
Should we add a sentence in here to introduce the subject or is it clear enough as-is ?

Reply:
I added a proposal